docs/init.rst

```diff
@@ -348,6 +348,7 @@ If you need to set attributes on a frozen class, you'll have to resort to the :r
    >>> Frozen(1)
    Frozen(x=1, y=2)
 
+Note that you may not access the hash code of the object in ``__attrs_post__init__`` if ``cache_hash=True``
```

Comment:
Period at the end of the sentence please. :)

Maybe use stronger wording and say `*must not* access`?